packages/OmegaPrint-Tests.package/TestUI.class/instance/testOmegaPrintString.st

```diff
@@ -0,0 +1,10 @@
+button functionality
+testOmegaPrintString
+
+	|codeHolder |
+	codeHolder := CodeHolder new.
+	codeHolder contentsSymbol: #omegaPrint.
+	codeHolder toggleOmegaPrint.
+	self assert: codeHolder contentsSymbol equals: #source.
+	codeHolder toggleOmegaPrint.
+	self assert: codeHolder contentsSymbol equals: #omegaPrint
```

Comment:
Sollte man das nicht lieber in zei Tests auslagern?
Wir testen hier zwei Sachen, theoretisch könnte eine funktionieren und die andere nicht

Reply:
bin ich nicht der Meinung, weil es toggle heißt und wir quasi in beide Richtungen togglen

Reply:
Würde ich auch nicht sagen, der Test testet halt diese eine Methode

Reply:
Ich denke auch, dass eine Methode reicht